cicd/pipelines/setting-compute-resource-quota-for-openshift-pipelines.adoc

```diff
@@ -0,0 +1,37 @@
+[id="setting-compute-resource-quota-for-openshift-pipelines"]
+= Setting compute resource quota for OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+:context: setting-compute-resource-quota-for-pipelines
+
+toc::[]
+
+
+A `ResourceQuota` object in {pipelines-title} controls the total resource consumption per namespace. You can use it to limit the quantity of objects created in a namespace, based on the type of the object. In addition, you can specify a compute resource quota to restrict the total amount of compute resources consumed in a namespace. 
+
+However, you might want to limit the amount of compute resources consumed by pods resulting from a pipeline run, rather than setting quotas for the entire namespace. Currently, {pipelines-title} does not allow you to directly specify the compute resource quota for a pipeline. 
```

Comment:
```suggestion
However, you might want to limit the amount of compute resources consumed by pods resulting from a pipeline run, rather than setting quotas for the entire namespace. Currently, {pipelines-title} does not enable you to directly specify the compute resource quota for a pipeline. 
```
Search for "anthropomorphism" in the IBM SG.